README.md

```diff
@@ -193,50 +193,45 @@ Configuration is handled differently depending on how the connector is installed
+| client.http.open.timeout          | The timeout period for opening an HTTP connection.                                                                                                                                                                              | *Integer* | 4000    |
+| client.http.request.timeout       | The timeout period for any single HTTP request and response.                                                                                                                                                                    | *Integer* | 15000   |
+| client.proxy                      | Sets whether the configured proxy options are used.                                                                                                                                                                             | *Boolean* ||
+| client.proxy.host                 | The proxy host to use. Requires `client.proxy` to be set to `true`.                                                                                                                                                             | *String*  ||
+| client.proxy.non.proxy.hosts      | A list of hosts excluded from using the proxy. Requires `client.proxy` to be set to `true`.                                                                                                                                     | *List*    ||
+| client.proxy.username             | The client proxy username. Requires `client.proxy` to be set to `true`.                                                                                                                                                         | *String*  ||
+| client.proxy.password             | The client proxy password. Requires `client.proxy` to be set to `true`.                                                                                                                                                         | *String*  ||
+| client.proxy.port                 | The client proxy port. Requires `client.proxy` to be set to `true`.                                                                                                                                                             | *Integer* ||
+| client.proxy.pref.auth.type       | The authentication type to use with the client proxy. Must be one of `BASIC`, `DIGEST` or `X_ABLY_TOKEN`. Requires `client.proxy` to be set to `true`.                                                                          | *String*  | Basic   |
+| client.push.full.wait             | Sets whether Ably should wait for all the effects of push REST requests before responding.                                                                                                                                      | *Boolean* ||
```

Comment:
What's a "push REST request", and what effects will this wait for?

Reply:
Sorry, there's a fair bit of diff noise on this PR because editing this table in IntelliJ causes is to reformat the whole thing 🙄 

All of those configs were already there, I've deleted the ones that (I think) only impact realtime connections, as we're not using that anymore. 

This one is essentially just exposing `ClientOptions.pushFullWait` from the Java SDK, but I don't know what it does: https://github.com/ably/kafka-connect-ably/blob/integ_v3/src/main/java/com/ably/kafka/connect/config/ChannelSinkConnectorConfig.java#L295

Reply:
Ok, happy to pretend I never saw it 😅